mac/autobuild/codeQL/autobuild_mac_3_copy_files.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+# autobuild_3_copy_files: copy the built files to deploy folder
+
+
+
+echo "no action here"
```

Comment:
Is this file really needed?

Reply:
it is not necessary, but it helps stylewise.
it obviously isn't in terms of execution, but it makes the 3-step-structure visible & consistent.